packages/e2e/lib/e2e.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'dart:async';
 
+import 'package:mutex/mutex.dart';
```

Comment:
I believe a mutex is essentially the same as a pool of size 1. Since we already  have a dependency on the `pool` package (I believe `flutter_tool` depends on it), we can continue using it rather than introduce another dependency.

Reply:
You mean this package right:https://pub.dev/packages/pool (The one we used in [felt](https://github.com/flutter/engine/blob/master/lib/web_ui/dev/test_runner.dart#L517) :smile: )

Pool is more suitable when we have an operation where a Delegator distributes it to multiple Workers. Such as compiling tests with dart2js in web test harness. It is not suitable for used for mutual exclusion. Our case is basic resource sharing problem: where mutex controls access to a _critical section_.

Reply:
I realized, there is no way (at least I couldn't manage) to run the tests methods in parallel when we run these tests with `flutter drive`, therefore I removed the mutex :)